### PR TITLE
#11 moved entities to new application

### PR DESCRIPTION
**Pull request by elyutikov**
may need adjustment

### CODE REVIEW

File: new/src/main/scala/model/Attachment.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+case class Attachment(
+    id: Int,
+    fid: String,
+    messageId: Int,
+    userId: Int,
+    fileName: String,
+    size: Long,
+    isAvatar: Boolean,
+    mime: String,
+    timestamp: Timestamp,
```
Comment (Ssstlis) [inline new/src/main/scala/model/Attachment.scala]: I think we should led to using Long or OffsetDateTime

File: new/src/main/scala/model/Message.scala [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import java.sql.Timestamp
+
+case class Message(
+    id: Int,
+    from: Int,
+    key: String,
+    text: String,
+    isDeleted: Boolean,
+    threadId: Int,
+    quoteFor: Int,
```
Comment (Ssstlis) [inline new/src/main/scala/model/Message.scala]: I suggest this fields must be an optional values

File: new/src/main/scala/model/Message.scala [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    chatId: Int,
+    timestampPost: Timestamp,
+    timestampChange: Option[Timestamp] = None,
+    timestampDelivery: Option[Timestamp] = None,
+    timestampRead: Option[Timestamp] = None
```
Comment (Ssstlis) [inline new/src/main/scala/model/Message.scala]: Let's use Long or OffsetDateTime

File: new/src/main/scala/model/User.scala [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    firstName: String,
+    lastName: String,
+    surname: Option[String],
+    position: Option[String],
+    groupId: Option[Int],
+    password: String,
+    login: String,
+    isActive: Boolean,
+    joinDate: Timestamp,
+    lastActivity: Timestamp,
```
Comment (Ssstlis) [inline new/src/main/scala/model/User.scala]: Let's use Long or OffsetDateTime

File: new/src/main/scala/model/Group.scala [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package model
+
+import java.sql.Timestamp
+
+case class Group(
+    id: Int,
+    name: String,
+    ownerId: Int,
+    created: Timestamp,
```
Comment (Ssstlis) [inline new/src/main/scala/model/Group.scala]: Let's use Long or OffsetDateTime

File: new/src/main/scala/model/Attachment.scala [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+import java.sql.Timestamp
+
+case class Attachment(
+    id: Int,
+    fid: String,
+    messageId: Int,
+    userId: Int,
+    fileName: String,
+    size: Long,
+    isAvatar: Boolean,
```
Comment (Ssstlis) [inline new/src/main/scala/model/Attachment.scala]: I think we can set default value `false`

File: new/src/main/scala/model/Message.scala (hunk 2 above)
Comment (blessedvictim) [reply new/src/main/scala/model/Message.scala]: +

> I suggest this fields must be an optional values

+

Comment (blessedvictim) [reply new/src/main/scala/model/Message.scala]: `+`

**Pull request merged**